studies/RWM_error_3d_normal_high.csv

```diff
@@ -0,0 +1,200 @@
+1.107094319886217981e+00,5.652251302905041408e-01
```

Comment:
If we start doing things like this, we might need to set up a separate testing/development repo for them, so that normal users don't have to download all these files.

Reply:
I agree with not including file in the repository, its easy enough to have a separate "data" repo for these. Github can do something tricky for large file storage too, but I don't know much about this.